### PR TITLE
Move Earn pool info tab to accordion

### PR DESCRIPTION
**Pull request by jagodarybacka**
Resolves https://github.com/tallycash/extension/issues/1452
### What has been done
* changed contents of "Pool info" tab to match designs
* removed "Pool info" tab and created new accordion in the vault info section
* added doggo and eth icons as common, hardcoded icons in the SharedAssetIcon component

image

image

### CODE REVIEW

File: ui/components/Shared/SharedAssetIcon.tsx [hunk 1]
```diff
@@ -81,8 +82,9 @@ export default function SharedAssetIcon(props: Props): ReactElement {
           align-items: center;
           justify-content: center;
           ${hasHardcodedIcon
-            ? `background: url("${`./images/${symbol.toLowerCase()}@2x.png`}") center no-repeat;
-            background-size: 45% auto;`
+            ? `background: url("${`./images/assets/${symbol.toLowerCase()}.png`}");
```
Comment (henryboldi) [inline ui/components/Shared/SharedAssetIcon.tsx:85]: No more `@2x`?

File: ui/pages/Earn/AccordionAPR.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        <div className="label">If $DOGGO = $0.0016</div>
+        <div>{vault.APR?.low}</div>
+      </li>
+    </ul>
+    <style jsx>
+      {`
+        .container {
+          padding-bottom: 16px;
+        }
+        .content_row {
```
Comment (henryboldi) [inline ui/pages/Earn/AccordionAPR.tsx:79]: Prolly don't need a class here, right?

File: ui/pages/Earn/AccordionAPR.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    </style>
+  </div>
+)
+
+export const AccordionAPRContent = ({
+  vault,
+}: {
+  vault: AvailableVault
+}): ReactElement => (
+  <div className="container">
```
Comment (henryboldi) [inline ui/pages/Earn/AccordionAPR.tsx:41]: Is this `div.container` needed? Can prolly add the padding to `ul` below

File: ui/pages/EarnDeposit.tsx [hunk 4]
```diff
@@ -372,6 +299,14 @@ export default function EarnDeposit(): ReactElement {
               <div className="otherReward"> + {vault.asset.symbol}</div>
             </div>
           </li>
+          <li className="row np">
```
Comment (henryboldi) [inline ui/pages/EarnDeposit.tsx:302]: `np`?

File: ui/components/Shared/SharedAssetIcon.tsx (hunk 1 above)
Comment (jagodarybacka) [reply ui/components/Shared/SharedAssetIcon.tsx:85]: I've downloaded assets icons to their own folder so I don't think we need it, the only thing that matters is to have the same name as the asset symbol

File: ui/pages/EarnDeposit.tsx (hunk 4 above)
Comment (jagodarybacka) [reply ui/pages/EarnDeposit.tsx:302]: "no padding" I guess, will change it

File: ui/pages/Earn/AccordionPoolInfo.tsx [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    </div>
+    <style jsx>{`
+      .pool_container {
+        padding: 5px 10px 0;
+      }
+      .pool_row {
+        margin: 10px 0;
+        line-height: 24px;
+        flex-shrink: 1;
+        flex-grow: 0;
```
Comment (henryboldi) [inline ui/pages/Earn/AccordionPoolInfo.tsx:77]: I believe this should be `Green 20` according to the designs?

**Review by henryboldi — APPROVED** [review]
Excellent!! 👑

Added a new small nit, but it's certainly not blocking

**Pull request merged**